README.md

```diff
@@ -1 +1,168 @@
-# http_toolkit
\ No newline at end of file
+# http_toolkit
+The httptoolkit library is a tool for working with HTTP requests in Python that allows you to easily create, customize, and send requests to various services. It provides a simple interface for working with HTTP clients. The library also allows flexible transport customization and abstraction from a particular implementation of HTTP clients.
+
+## HTTPX_Service
+
+If you don't need to use a configured transport, use HTTPXService (for async -> AsyncHttpxService)
+
+```python
+from httptoolkit import HttpxService
+from httptoolkit import Header
+
+headers = (
+    Header(name="My-Header", value="my-value", is_sensitive=False),
+)
+httpbin = HttpxService("http://httpbin.org", headers=headers)
+httpbin.get("/get")
+httpbin.post("/post")
+```
+
+## Service
+
+If you want to use a configured transport, use Service. (Service -> HttpxTransport, AsyncService -> AsyncHttpxTransport)
+```python
+### Sync
+
+from httptoolkit import Service, Header
+from httptoolkit.transport import HttpxTransport
+
+
+class DummyService(Service):
+    pass
+
+
+DummyService(
+    headers=(Header(name="ServiceHeader", value="service-header", is_sensitive=False),),
+    transport=HttpxTransport(base_url="https://example.com:4321", proxies={"http://": "http://10.10.1.10:3128"}),
+    ## base_url in this case is passed to transport
+)
+```
+
+```python
+### Async
+
+from httptoolkit import AsyncService, Header
+from httptoolkit.transport import AsyncHttpxTransport
+
+
+class DummyService(AsyncService):
+    pass
+
+
+DummyService(
+    headers=(Header(name="ServiceHeader", value="service-header", is_sensitive=False),),
+    transport=AsyncHttpxTransport(base_url="https://example.com:4321", proxies={"http://": "http://10.10.1.10:3128"}),
+    ## base_url in this case is passed to transport
+)
+```
+
+### Sending a request
+
+```python
+### Async
```

Comment:
It seems there are no async examples below

Reply:
Yeah, I think I will remove this line